ansibullbot/utils/moduletools.py

```diff
@@ -418,13 +469,102 @@ def last_commit_for_file(self, filepath):
         return so.strip()
 
     def get_module_blames(self):
+
+        logging.debug('build email cache')
+        emails_cache = self.session.query(Email)
+        emails_cache = [(x.email,x.login) for x in emails_cache]
+        self.emails_cache = dict(emails_cache)
```

Comment:
`self.emailmap` must be removed (because being replaced by `self.emails_cache`).

In `extract_github_id` method, `self.emails_cache` must be used (instead of `self.emailmap`).